src/todo.spec.js

```diff
@@ -1,5 +1,4 @@
-import { jest } from '@jest/globals';
-import { add, format, formatList, list } from './todo.js';
+import { jest } from '@jest/globals'; import { add, findById, format, formatList, list } from './todo.js';
```

Comment:
This should be separated

Reply:
true true didnt mean to put that in one line